src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
-body {
+$product-font-color: #060b35;
+
+@import 'stars';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.product-cards-container {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.product-card {
```

Comment:
move `product-card` block styles to an own file `product.scss`
So you will have `index.scss`, `product-card.scss`, `stars.scss`, `variables.scss`, `common.scss` files